evo-delivery-app/evo-delivery-backend/src/controllers/driver.controller.ts

```diff
@@ -4,11 +4,12 @@ import { Driver } from '../types';
 import { INVALID, OK } from '../utils/response.utils';
 
 export const getDrivers = async (req: Request, res: Response) => {
-  const page = req.params.page || 1;
-  const limit = req.params.limit || 10;
+  const page :number= parseInt(req.query.page as string) || 1;
+  const limit :number= parseInt(req.query.limit as string) || 100;
```

Comment:
Let's consider using the unary operator `+` or `Number()` for parsing because it can make an issue when the value is not well defined as int.

For example: 
An empty string "" evaluates to a 0, while parseInt evaluates it to NaN. 
see [this](https://stackoverflow.com/questions/17106681/parseint-vs-unary-plus-when-to-use-which)

Example:

```
+req.query.page
```
or
```
Number(req.query.page)
```

I would like to hear your thoughts about it

Reply:
i agree with you, i'll change it